jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/writer/EncodingWriter.java

```diff
@@ -11,41 +11,45 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.ee10.servlet.writer;
+package org.eclipse.jetty.io.writer;
 
 import java.io.IOException;
+import java.io.OutputStream;
 import java.io.OutputStreamWriter;
-import java.io.UnsupportedEncodingException;
 import java.io.Writer;
-
-import org.eclipse.jetty.ee10.servlet.HttpOutput;
+import java.nio.charset.Charset;
 
 /**
- *
+ * An implementation of {@link AbstractOutputStreamWriter} that internally
+ * uses {@link java.io.OutputStreamWriter}.
  */
-public class EncodingHttpWriter extends HttpWriter
+public class EncodingWriter extends AbstractOutputStreamWriter
```

Comment:
This class should be internal or package local, or an private inner class of `AbstractOutputStreamWriter`.

Reply:
I can make them all internal if I have a good place for the `newWriter(Charset)` static method?   Is `Content` OK for that?  kind of???   I'll put it there for now, move classes to internal and we can evaluate.

Reply:
Ah the AbstractOutputStreamWriter class needs to be visible so we can unwrap to find the output stream (for async close in servlets).  So static methods can live there